code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -118,6 +118,9 @@
 					emote_see = list("shakes its head", "shivers")
 					desc = "It's a corgi."
 					set_light(0)
+					flags = 0
+					atmos_requirements = list("min_oxy" = 5, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
+					minbodytemp = 250

```

Comment:
When I do that, the atmos_requirements and minbodytemp vars seem to lack any sort of value after taking off the hardsuit.